### ISSUE TITLE
markdown.vue文件引入js问题,

### ISSUE THREAD

**Issue by xuxuzhaozhao** [ovenslove/vue-mdEditor#13]
大哥帮看下,markdown.vue文件引入这两个JS文件(`import hljs from '../../../static/js/highlight.min.js'
    import range from '../../../static/js/rangeFn.js'`)的时候,总是报
`Module parse failed: Unexpected token (7:0)
You may need an appropriate loader to handle this file type.`

**Comment by hanLiang920** [comment]
请问下解决了吗？我也遇到这个问题了

**Comment by wait-for-the-wind** [comment]
是的  是引入高亮JS  `import hljs from '../../assets/markdown/js/highlight.min.js' `  出现了问题，不知道问题在哪？现在解决了吗
